index.src.html

```diff
@@ -869,9 +980,14 @@ <h3 id="information-leakage">Information Leakage</h3>
   <a>"<code>unsafe-url</code>"</a> might leak the origin and the URL of
   a secure site respectively via insecure transport.
 
-  Those two policies are include in the spec nevertheless to lower the friction
+  Those two policies are included in the spec nevertheless to lower the friction
   of sites adopting secure transport.
 
+  Authors can use the policy states <a>"<code>safe-origin</code>"</a>,
+  <a>"<code>safe-origin-when-cross-origin</code>"</a> and
+  <a>"<code>no-referrer</code>"</a> to prevent sensitive information from
```

Comment:
This section seems a little misleading, since I think every policy except `unsafe-url` could be used in some way to prevent sensitive information from leaking. For example, `origin` is a policy that can be used to protect capability URLs. Are there particular privacy properties of `safe-origin` and `safe-origin-when-cross-origin` that you're trying to call out here?